.travis.yml

```diff
@@ -38,8 +38,9 @@ env:
   - VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB" PG_LOGS_ENABLE="1" PG_NON_DURABILITY="1"  # ODOO_REPO usage example
   # - VERSION="6.1" INCLUDE="test_module,second_module"
   - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="27" TRAVIS_PULL_REQUEST="false" # Use main pylint config file
-  - VERSION="master" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="84" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
-  - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="73" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
+  - VERSION="master" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="111" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
+  - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="100" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
+  - LINT_CHECK="1" TESTS="0" PYLINT_ODOO_JSLINTRC="DFTL_JSLINTRC" PYLINT_EXPECTED_ERRORS="111"
```

Comment:
You need download and use it
You can use a wget:
https://github.com/Vauxoo/maintainer-quality-tools/pull/131/files/8220900964d67c349038b193cd32b20fcbb43234#diff-17b9133d6cebf66784dc392176486372R15

An you can use the same path to set the environment variable